cwf/gateway/integ_tests/gy_enforcement_test.go

```diff
@@ -491,7 +491,7 @@ func TestGyCreditExhaustionRedirect(t *testing.T) {
 
 	// Check that enforcement stats flow was not removed and data was passed
 	assert.Eventually(t,
-		tr.WaitForEnforcementStatsForRuleGreaterThan(imsi, "static-pass-all-ocs2", 1*MegaBytes), time.Minute, 2*time.Second)
+		tr.WaitForEnforcementStatsForRuleGreaterThan(imsi, "static-pass-all-ocs2", 0.25*MegaBytes), time.Minute, 2*time.Second)
 
 	// When we initiate a UE disconnect, we expect a terminate request to go up
```

Comment:
The tests seems to missing a check that not redirect rule is present in this second case. Could you try to add one?

Reply:
I added a check that it does in my latest commit. I had to manually implement this, but effectively we can assert this by waiting for a minute that no redirect rule appears for a minute.

Reply:
just wait for 15sec make sure the rule is not there

Reply:
:+1: